model/instance/init.go

```diff
@@ -4,6 +4,18 @@ import "github.com/cozy/cozy-stack/pkg/config/config"
 
 var service *InstanceService
 
+// Service handle all the interactions with the "instance" domaine.
+//
+// This interface has several implementations:
+// - [InstanceService] with the business logic
+// - [Mock] with a mock implementation
+type Service interface {
+	Get(domain string) (*Instance, error)
+	GetFromCouch(domain string) (*Instance, error)
```

Comment:
It looks like this method has been renamed to `GetWithoutCache`

Reply:
True, I fixed and add a test to check the implementation.